SPECS-CROSS/gcc-bootstrap_cross/gcc-bootstrap2_cross.spec

```diff
@@ -0,0 +1,440 @@
+%global security_hardening nofortify
+%global debug_package %{nil}
+%global set_build_flags %{nil}
+
+# Globals which should be in a macro file.
+# These should be set programatically in the future.
+%global _host_arch      x86_64
+%global _target_arch    aarch64
+
+%global _tuple          %{_target_arch}-%{_vendor}-linux-gnu
+%global _cross_name     %{_target_arch}-%{_vendor}-linux-gnu
+
+# Folders which should be in our macro file
+%global _opt                /opt/
+%global _crossdir           /opt/cross/
+
+# Generally we include '/usr' in most paths.
+# Can we also use '/usr' for our paths? This will bring us in line with the
+# %%configure macro which sets these.
+%global _bindir            /bin
+%global _sbindir           /sbin
+%global _libdir            /lib
+%global _lib64dir          /lib64
+%global _libexecdir        /libexec
+%global _datadir           /share
+%global _docdir            /share/doc
+%global _includedir        /include
+%global _infodir           /share/info
+%global _mandir            /share/man
+%global _oldincludedir     /include
```

Comment:
How were you envisioning using `_oldincludedir` vs `_includedir`?

Reply:
We need to remove this work around (actually set the /usr paths correctly when calling ./configure). See %configure in the macros file here: https://github.com/microsoft/CBL-Mariner/blob/5bc8fa1c8f0a8c8c6bd4b012a0072cbf4fd68f3e/SPECS/mariner-rpm-macros/macros#L70